backend/azure/file.go

```diff
@@ -29,7 +30,7 @@ type File struct {
 func (f *File) Close() error {
 	if f.tempFile != nil {
 		defer func() {
-			f.tempFile.Close()
+			_ = f.tempFile.Close()
```

Comment:
we would never want to do anything with these errors?

Reply:
good question... here i'm not changing any behavior except making it explicit what we're doing.  Close() is really the hardest thing to do right.  Either you run Close() in a defer or you rerun it at every exit point of the function.  But each exit point of the function either returns an error already or nil (no error).  Unfortunately, we can't return a value (or error) from inside defer.  Furthermore, we can't really return 2 errors without concatenating them somehow.  
The particular quandary of handling errors on Close() doesn't really seem to have a community standard or idiomatic approach.  We made a stab at it early using a error type that allowed us to append errors to AND utilized named error returns so that both the normal code and Close() defer closure would update the error.  You can see the code at https://github.com/C2FO/vfs/blob/8558f12c8f4cb571cd32a610936f0c1d48aa737b/errors.go and example usage at https://github.com/C2FO/vfs/blob/8558f12c8f4cb571cd32a610936f0c1d48aa737b/errors_example_test.go.  However we soon found this a cumbersome and kludgy solution with very little gain.  Close() is the one return value that we regularly ignore.
I'm all for a better solution though!